MathlibExtras/DotNotation.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2023 Martin Dvorak. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Martin Dvorak, Kyle Miller
```

Comment:
```suggestion
Authors: Martin Dvorak
```
No need to list me as an author -- the authors list is generally meant to be who takes some responsibility for the module, and while I'm happy to support you in helping to write the occasional unexpander, I'm probably not going to maintain this module myself. If you're worried about attribution, you can add me in the copyright line, but it's not necessary.